cmd/remove.go

```diff
@@ -82,8 +82,8 @@ func init() {
 	}
 	rootCmd.AddCommand(deleteCmd)
 
-	deleteCmd.PersistentFlags().BoolVar(&raw.recursive, "recursive", false, "look into sub-directories recursively when syncing between directories.")
-	deleteCmd.PersistentFlags().StringVar(&raw.logVerbosity, "log-level", "INFO", "define the log verbosity for the log file, available levels: INFO(all requests/responses), WARNING(slow responses), ERROR(only failed requests), and NONE(no output logs).")
-	deleteCmd.PersistentFlags().StringVar(&raw.include, "include", "", "only include files whose name matches the pattern list. Example: *.jpg;*.pdf;exactName")
-	deleteCmd.PersistentFlags().StringVar(&raw.exclude, "exclude", "", "exclude files whose name matches the pattern list. Example: *.jpg;*.pdf;exactName")
+	deleteCmd.PersistentFlags().BoolVar(&raw.recursive, "recursive", false, "Look into sub-directories recursively when syncing between directories.")
+	deleteCmd.PersistentFlags().StringVar(&raw.logVerbosity, "log-level", "INFO", "Define the log verbosity for the log file. Available levels include: INFO(all requests/responses), WARNING(slow responses), ERROR(only failed requests), and NONE(no output logs). (default 'INFO')")
+	deleteCmd.PersistentFlags().StringVar(&raw.include, "include", "", "Only include files where the name matches the pattern list. For example: .jpg;.pdf;exactName")
+	deleteCmd.PersistentFlags().StringVar(&raw.exclude, "exclude", "", "Exclude files where the name matches the pattern list. For example: .jpg;.pdf;exactName")
```

Comment:
>.jpg;.pdf;exactName [](start = 137, length = 19)

In our include and exclude filters, we make use of path.Match. This means that we'll listen to wildcards, and that these would just match to files exactly named .jpg, .pdf, and exactName.

Please keep the wildcards, otherwise this is invalid documentation. If this is formatting as markdown, manual edits may have to be made when generated.